### DISCUSSION TITLE
Pre release Q&A

### DISCUSSION THREAD

**Discussion by kakajansh** [kakajansh/echo#45] [Q&A]
Hi devs,

I am about to release new version with support of **null safety** and switching to [pusher_client](https://pub.dev/packages/pusher_client) library. Could you please share some usage info?

- Which broadcaster type do you use `pusher` or `socket.io`?
- What do you use for backend?  
   - [laravel-echo-server](https://github.com/tlaverdure/laravel-echo-server)
   - [laravel-websockets](https://github.com/beyondcode/laravel-websockets)
   - [pusher](https://pusher.com/)
   - any other?

**Comment by nicolasvahidzein** [comment]
Wonderful news buddy. 
I use pusher and laravel websockets.

**Comment by PedroCM-git** [comment]
Thats neat.

I use pusher in both.
